modules/comment/comment.graphql.js

```diff
@@ -12,9 +12,9 @@ show: Boolean
 const commentsInput = `
 input commentsInput {
```

Comment:
Please, rename it to `CommentInput`. For the future, all inputs should be named in a format `${ModelName}Input`

Reply:
Renamed